lighthouse-core/report/html/renderer/util.js

```diff
@@ -425,9 +429,10 @@ class Util {
     }
 
     // TODO(paulirish): revise Runtime Settings strings: https://github.com/GoogleChrome/lighthouse/pull/11796
-    const deviceEmulation = settings.formFactor === 'mobile'
-      ? Util.i18n.strings.runtimeMobileEmulation
-      : Util.i18n.strings.runtimeDesktopEmulation;
+    const deviceEmulation = {
```

Comment:
The fallback case shouldn't ever happen for the `'mobile'|'desktop'` type it is today, so probably worth adding a comment about why there's a fallback to `runtimeNoEmulation`